_news-items/2020-04-07-launch.html

```diff
@@ -0,0 +1,39 @@
+---
+# Copyright 2020 seL4 Project a Series of LF Projects, LLC.
+# SPDX-License-Identifier: CC-BY-SA-4.0
```

Comment:
I think I would add all files in _news-items to `.reuse/dep5` instead, that would avoid a lot of duplication and make it a bit cleaner overall.

Reply:
I don't think we should do that. The `dep5` is an emergency fall back for when nothing else really works, and it does work nicely here.